README.md

```diff
@@ -109,44 +102,32 @@ Here is a more typical example using docker-composer (formerly known as fig):
 For ``docker-compose.yml``:
 
 ```
-db:
-  image: kartoza/postgis:9.4-2.1
-  volumes:
-    - ./pg/postgres_data:/var/lib/postgresql
-    - ./pg/setup_data:/home/setup
-  environment:
-    - USERNAME=docker
-    - PASS=docker
-
-dbbackups:
-  image: kartoza/pg-backup:9.4
-  hostname: pg-backups
+sftpbackup:
+  image: kartoza/sftp-backup
+  hostname: sftp-backup
   volumes:
     - ./backups:/backups
-  links:
-    - db:db
+    - ./target:/target
   environment:
-    - DUMPPREFIX=PG_YOURSITE
-    # These are all defaults anyway, but setting explicitly in
-    # case we ever want to ever use different credentials
-    - PGUSER=docker
-    - PGPASSWORD=docker
-    - PGPORT=5432
-    - PGHOST=db
-    - PGDATABASE=gis  
+    - USE_SFTP_BACKUP=True
```

Comment:
I think it is nice to put in the defaults for all env vars that can be passed to the container (as I did for the PG example you based this service on) - it makes it very quick and obvious what env vars you can change